### DISCUSSION TITLE
Demo source

### DISCUSSION THREAD

**Discussion by villagra** [steveruizok/perfect-freehand#41] [Q&A]
Hi! 
First of all, thanks a lot for this! 🙇🙇 
I have just a small question, is the demo page (https://perfect-freehand-example.vercel.app/) source code available? I'm investigating about the possibility to integrate this into a pet project and that code would be very helpful. 

Thanks a lot!

**Comment by steveruizok** [comment]
Yep, it's in the packages/dev folder!
